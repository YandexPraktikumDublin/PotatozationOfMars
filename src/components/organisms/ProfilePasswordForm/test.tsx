import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { ProfilePasswordForm } from '.'

describe('<ProfilePasswordForm />', () => {
  it('should renders correct <ProfilePasswordForm />', () => {
    const successCallback = jest.fn()

    const wrapper = shallow(
      <ProfilePasswordForm successCallback={successCallback} />
    )

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
