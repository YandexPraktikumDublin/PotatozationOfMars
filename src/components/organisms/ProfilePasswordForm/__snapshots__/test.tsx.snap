// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfilePasswordForm /> should renders correct <ProfilePasswordForm /> 1`] = `
<BaseForm
  buttonText="Change password"
  formError=""
  initialValues={
    Object {
      "newPassword": "",
      "oldPassword": "",
    }
  }
  onSubmit={[Function]}
  validationSchema={
    ObjectSchema {
      "_blacklist": ReferenceSet {
        "list": Set {},
        "refs": Map {},
      },
      "_blacklistError": undefined,
      "_excludedEdges": Array [],
      "_nodes": Array [
        "newPassword",
        "oldPassword",
      ],
      "_sortErrors": [Function],
      "_typeError": [Function],
      "_whitelist": ReferenceSet {
        "list": Set {},
        "refs": Map {},
      },
      "_whitelistError": undefined,
      "conditions": Array [],
      "deps": Array [],
      "exclusiveTests": Object {},
      "fields": Object {
        "newPassword": StringSchema {
          "_blacklist": ReferenceSet {
            "list": Set {},
            "refs": Map {},
          },
          "_blacklistError": undefined,
          "_mutate": undefined,
          "_typeError": [Function],
          "_whitelist": ReferenceSet {
            "list": Set {},
            "refs": Map {},
          },
          "_whitelistError": undefined,
          "conditions": Array [],
          "deps": Array [],
          "exclusiveTests": Object {
            "min": true,
            "required": true,
          },
          "spec": Object {
            "abortEarly": true,
            "label": undefined,
            "meta": undefined,
            "nullable": false,
            "presence": "required",
            "recursive": true,
            "strict": false,
            "strip": false,
          },
          "tests": Array [
            [Function],
            [Function],
          ],
          "transforms": Array [
            [Function],
          ],
          "type": "string",
        },
        "oldPassword": StringSchema {
          "_blacklist": ReferenceSet {
            "list": Set {},
            "refs": Map {},
          },
          "_blacklistError": undefined,
          "_mutate": undefined,
          "_typeError": [Function],
          "_whitelist": ReferenceSet {
            "list": Set {},
            "refs": Map {},
          },
          "_whitelistError": undefined,
          "conditions": Array [],
          "deps": Array [],
          "exclusiveTests": Object {
            "min": true,
            "required": true,
          },
          "spec": Object {
            "abortEarly": true,
            "label": undefined,
            "meta": undefined,
            "nullable": false,
            "presence": "required",
            "recursive": true,
            "strict": false,
            "strip": false,
          },
          "tests": Array [
            [Function],
            [Function],
          ],
          "transforms": Array [
            [Function],
          ],
          "type": "string",
        },
      },
      "spec": Object {
        "abortEarly": true,
        "label": undefined,
        "meta": undefined,
        "nullable": false,
        "presence": "optional",
        "recursive": true,
        "strict": false,
        "strip": false,
      },
      "tests": Array [],
      "transforms": Array [
        [Function],
      ],
      "type": "object",
    }
  }
>
  <BaseInput
    autoComplete="on"
    name="oldPassword"
    placeholder="Old password"
    type="password"
  />
  <BaseInput
    autoComplete="on"
    name="newPassword"
    placeholder="New password"
    type="password"
  />
</BaseForm>
`;
